docs/vereisten/automatische_logregistratie.md

```diff
@@ -28,12 +29,23 @@ Het doel van deze registratie is om de transparantie en verantwoordingsplicht va
 
 Voor AI-systemen met een hoog-risico voorziet de loggingcapaciteit ten minste in: a) de registratie van de duur van elk gebruik van het systeem; b) de referentiedatabank aan de hand waarvan de inputdata zijn gecontroleerd door het systeem; c) de inputdata ten aanzien waarvan de zoekopdracht een match heeft opgeleverd; d) de identificatie van natuurlijke personen die betrokken zijn bij de verificatie van de resultaten.
 
+Voor AI-systemen die door bestuursorganen worden gebruikt of AI-systmen die persoonsgegevens verwerken leveren de BIO en AVG vergelijkbare verplichingen op die ook van toepassing zijn op AI-systmen die niet gezien worden als een AI-systeem met hoog risico. Daarbij komen nog verplichtingen om de logs doorlopend of periodiek te monitoren op incidenten.
 
 ## Bronnen
 
 | Bron                        |
 |-----------------------------|
 |Artikel 12 Registratie- AI verordening|
+|Baseline Informatiebeveiliging Overheid hoofdstuk 12.4|
+|Artikel 5 en 32 AVG|
+
+## Wanneer van toepassing?
+Op AI-systemen met hoog risico, AI-systmeen onder de verantwoordelijkheid van bestuursorganen en AI-systemen die persoonsgegevens verwerken.
```

Comment:
```suggestion

```

Reply:
Dit hoofdstuk krijgt nog een andere invulling, afhankelijk van de beslisboom. Wordt vervolgd dus. Dit willen we deels automatiseren, zodat we ook de functionaliteit krijgen om automatisch vereisten op te halen die horen bij jouw situatie.